src/lib/utils/getHighlightFromSiblings.ts

```diff
@@ -0,0 +1,20 @@
+import unescape from './unescape';
+import { HighlightedParts } from '../../types';
+
+const getHighlightFromSiblings = (parts: HighlightedParts[], i: number) => {
```

Comment:
this could be `export default getHighlightFromSiblings(parts: HighlightedParts[], i: number) {` which has a bit less syntax. Not really worried about it though

Reply:
Isn't it better to do named export? I know that we don't do it in instantsearch.js but just wondering